packages/web3-eth/src/utils/transaction_builder.ts

```diff
@@ -130,7 +128,7 @@ export async function defaultTransactionBuilder<ReturnType = Transaction>(option
 	privateKey?: HexString | Buffer;
 }): Promise<ReturnType> {
 	// let populatedTransaction = { ...options.transaction } as unknown as InternalTransaction;
-	let populatedTransaction = format(
+	const populatedTransaction = format(
```

Comment:
was getting liner error
'populatedTransaction' is never reassigned. Use 'const' instead  prefer-const